static-assets/components/cstudio-dashboard-widgets/go-live-queue.js

```diff
@@ -323,7 +323,7 @@ CStudioAuthoringWidgets.GoLiveQueueDashboard = function (widgetId, pageId) {
    */
   this.handleTopDownPathDependenciesItemsClick = function (event, matchedEl) {
     var isChecked = matchedEl.checked;
-    var selectedElementURI = matchedEl.id;
+    var selectedElementURI = decodeURIComponent(matchedEl.id);
```

Comment:
This fixes the console error but reject dialog still doesn't render the selected items on the dialog.